### PR TITLE
[CoreBundle] Add sylius_order_payment missing transitions

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

The `from` parts of `partially_pay` and `partially_refund` were missing :)

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml [hunk 1]
```diff
@@ -25,6 +26,7 @@ winzou_state_machine:
                 from: [awaiting_payment, partially_paid]
                 to: paid
             partially_refund:
+                from: [paid, partially_refunded]
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml]: What about `partially_paid`?

Comment (TheMadeleine) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml]: Should we add it?

Comment (TheMadeleine) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml]: While analysing it for the graph I've assumed it is needed :)

**Comment by pjedrzejewski** [comment]
👍

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml [hunk 2]
```diff
@@ -25,9 +26,10 @@ winzou_state_machine:
                 from: [awaiting_payment, partially_paid]
                 to: paid
             partially_refund:
+                from: [paid, partially_paid, partially_refunded]
                 to: partially_refunded
             refund:
-                from: [paid, partially_refunded]
+                from: [paid,, partially_paid, partially_refunded]
```
Comment (venyii) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml]: Is it supposed to have two commas here? `paid,, partially_paid`

Comment (TheMadeleine) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml]: Of course not :<

**Pull request merged**